packages/block-library/src/gallery/gallery-image.js

```diff
@@ -103,6 +119,22 @@ class GalleryImage extends Component {
 		}
 	}
 
+	/**
+	 * Note that, unlike the DOM, all React events bubble,
+	 * so this will be called after the onBlur event of any figure's children.
+	 */
+	onBlur() {
+		this.debouncedOnDeselect();
+	}
+
+	/**
+	 * Note that, unlike the DOM, all React events bubble,
+	 * so this will be called after the onBlur event of any figure's children.
```

Comment:
np: did you mean `s/onBlur/onFocus/` in this comment?

Reply:
```suggestion
	 * so this will be called after the onFocus event of any figure's children.
```